doc/source/data/loading-data.rst

```diff
@@ -919,6 +919,31 @@ run against the collection.
         collection="my_collection",
     )
 
+Reading Iceberg
+~~~~~~~~~~~~~~~
+
+To read data from Iceberg, call :func:`~ray.data.read_iceberg` and specify the catalog ID, namespace (database), and table.
+To read from Iceberg, install the `Python Client for Iceberg <https://py.iceberg.apache.org/>`_.
```

Comment:
"To read data from Iceberg" and "To read from Iceberg" feels repeating. could you rephrase?